sonar-orchestrator-locators/src/test/java/com/sonar/orchestrator/locator/MavenArtifactoryTest.java

```diff
@@ -87,8 +88,8 @@ public void downloadToFile_whenSuccessful() throws Exception {
     assertThat(targetFile).exists().hasContent("this_is_bytecode");
 
     RecordedRequest request = mockWebServerRule.getServer().takeRequest();
-    assertThat(request.getTarget()).isEqualTo("/org/sonarsource/sonarqube/sonar-plugin-api/3.0/sonar-plugin-api-3.0.jar");
-    assertThat(request.getHeaders().get("Authorization")).isNull();
+    Assertions.assertThat(request.getTarget()).isEqualTo("/org/sonarsource/sonarqube/sonar-plugin-api/3.0/sonar-plugin-api-3.0.jar");
+    Assertions.assertThat(request.getHeaders().get("Authorization")).isNull();
```

Comment:
Similar remark about the changes in this test, the `assertThat` static import is already present so we could use it instead